packages/flutter/test/material/carousel_test.dart

```diff
@@ -1413,6 +1413,34 @@ void main() {
       expect(buttonPressed, isTrue);
     },
   );
+
+  testWidgets('CarouselView does not crash if layout constraints are zero', (
```

Comment:
Might as well add a comment here:

```suggestion

  // Regression test for https://github.com/flutter/flutter/issues/160679.
  testWidgets('CarouselView does not crash if layout constraints are zero', (
```

Reply:
Done